figures/sources/class-diagram.puml

```diff
@@ -15,7 +15,8 @@ skinparam MinClassWidth 220
 Class DAPTScript as "**DAPT Script**" [[#dapt-script]] {
     Workflow Type [[[#workflow-type]]]
     Script Type [[[#script-type]]]
-    Primary Language [[[#primary-language]]]
+    Default Language [[[#default-language]]]
```

Comment:
Why did we change from Primary to Default? I did prefer Primary but won't insist if you still prefer "Default"

Reply:
My concern is that Primary has connotations that could lead to misunderstanding. Primary (also the alternative "Initial") suggests some sort of time or precedence ordering, as though the language specified here is more important, or the starting language for transcription etc. whereas Default is more neutral in that respect, and feels more specific to this particular instance of the document.

Reply:
I do think the language specified here is more important. The note even says "a language arbitrarily chosen by the author".
My concern with "default" is that it is an overloaded term. We use "default value" in plenty of places to refer to the value when not specified.
Could we just say "Language"?

Reply:
I'm going to raise this as a separate follow-on issue.

Reply:
Raised as #204